README.md

```diff
@@ -10,23 +10,52 @@ pipenv shell
 ### CLI
 ```
 python stats.py --help
-usage: stats.py [-h] -at ACCESS_TOKEN [-m MODE] -u USERS [USERS ...]
+usage: stats.py [-h] -at ACCESS_TOKEN [-m leaderboard, prs, all] -u USERS [USERS ...]
                 [-s START_DATE] [-e END_DATE]
 
+arguments:
+  -at ACCESS_TOKEN, --access-token ACCESS_TOKEN
+  -u USERS [USERS ...], --users USERS [USERS ...]
+
 optional arguments:
   -h, --help            show this help message and exit
-  -at ACCESS_TOKEN, --access-token ACCESS_TOKEN
   -m MODE, --mode MODE
-  -u USERS [USERS ...], --users USERS [USERS ...]
+                        one of leaderboard, prs or all
   -s START_DATE, --start-date START_DATE
                         date format YYYY-mm-dd
   -e END_DATE, --end-date END_DATE
                         date format YYYY-mm-dd
+                        
+example:
+python stats.py \
+-at f1a6f157cff1d9aa284350a71a813a25788f3be3 \
```

Comment:
@glacion is this your personal key? if so please reset it immediately

Reply:
@heaven00 No, it is not my personal key.

Reply:
👍